db/re/skill_db.yml

```diff
@@ -23717,11 +23715,11 @@ Body:
         Time: 15000
     Duration2:
       - Level: 1
-        Time: 4000
+        Time: 2000
       - Level: 2
-        Time: 8000
+        Time: 6000
       - Level: 3
-        Time: 12000
+        Time: 10000
```

Comment:
I wouldn't be surprised if this was actually 4s*level + fixed duration. But again just guessing.

Reply:
Confirmed.